package/yast2-storage-ng.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Wed Jul 12 15:37:16 UTC 2017 - ancor@suse.com
+
+- Replaced the original prototype of the expert partitioner with
+  the one coming from yast2-partitioner.
+- Removed Y2Storage::DevicesLists and not longer needed modules
+  from Y2Storage::Refinements
```

Comment:
I can add it but... are we planning to document in the changelog every single method added to the Y2Storage classes? It made sense (together with the version increase) when a separate y2-partitioner package needed them. But...